src/Compilers/VisualBasic/Portable/Syntax/SyntaxFactory.vb

```diff
@@ -1067,7 +1067,6 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             If options.Kind = SourceCodeKind.Regular Then
                 Throw New ArgumentException(VBResources.SyntaxTreeIsNotASubmission)
             End If
-
             Dim languageVersion As LanguageVersion = options.LanguageVersion
```

Comment:
>Dim languageVersion As LanguageVersion = options.LanguageVersion [](start = 12, length = 64)

Please revert this file to its original state, the only change is a removal of an empty line that should be restored.